### DISCUSSION TITLE
How to modify keycloak admin gui ?

### DISCUSSION THREAD

**Discussion by katariakhyati** [keycloak/keycloak#29226] [Archive]
I would like to modify the Keycloak Admin GUI by either changing the content of the welcome tab or removing the entire tab. However, I don't have direct access to the Dashboard.tsx file. Are there any options available to achieve this? I have tried to modify jar but no luck.
Any suggestions would be greatly appreciated.

**Comment by codespearhead** [comment]
@katariakhyati this is technically a duplicate of #28282 . Can you close this discussion in favor of that one? Since we're still working on centralizing related problems, I recommend reading #29044 , which describes symptoms of the original problem that are more similar to yours.

**Comment by katariakhyati** [comment]
Thanks for the inputs I will follow #29044.

**Comment by katariakhyati** [comment]
Re opening the discussion here as above mentioned tickets looks different from what I need to do.
Thanks

**Reply by RobertBasile-1** [reply]
I have looked into the previous mentioned tickets, to re iterate we want to modify the actual keycloak admin gui package with keycloak (opt/keycloak/lib/lib DIRECTORY). More specifically, https://github.com/jonkoops/keycloak-admin-ui/blob/main/src/dashboard/Dashboard.tsx we need to make modifications in the tabs, we note that this is the jar that is responsible to render the admin gui: org.keycloak.keycloak-admin-ui-22.0.5.jar. 

What we have tried is to modify the MAP file, including js file, but this was unsuccessful. We know that we dont have access to those tsx files as its provided from keycloak, but we would like to modify them.  @codespearhead
